sdk/metricsadvisor/ai-metrics-advisor/src/models.ts

```diff
@@ -1119,6 +1161,10 @@ export interface AnomalyAlertConfiguration {
    * Anomaly alerting configurations
    */
   metricAlertConfigurations: MetricAlertConfiguration[];
+  /**
+   * dimensions used to split alert
+   */
+  splitAlertByDimensions?: string[];
```

Comment:
`dimensionsToSplitAlerts` better?

Reply:
Will follow up on this after discussing with crew. Otherwise we'll make this change for JS only